src/types/positions.ts

```diff
@@ -0,0 +1,3 @@
+import { DialogPosition as DialogPositions } from "../constants";
+
+export type DialogPosition = DialogPositions;
```

Comment:
Not sure about the effort or the risk but would it be possible to define the type in the Dialog and export the type and not the const?